extensions/amp-lightbox-viewer/0.1/service/lightbox-manager-impl.js

```diff
@@ -159,4 +159,38 @@ export class LightboxManager {
       }
     });
   }
+
+  /**
+   * Find or create thumbnails for lightboxed elements.
+   * Return a list of thumbnails obj for lightbox gallery view
+   * The function is not implemented yet. Fake it for testing
+   * @return {!Array<{string,Element}>}
+   */
+  getThumbnails() {
+    const thumbnailList = [];
+    for (let i = 0; i < this.elements_.length; i++) {
+      const thumbnail = {
+        url: this.getThumbnailUrl_(this.elements_[i], i),
+        element: this.elements_[i],
+      };
+      thumbnailList.push(thumbnail);
+    }
+    return thumbnailList;
+  }
+
+  /**
+   * Get thumbnail url for single element.
+   * The function is not implemented yet. Fake for testing.
+   * @param {!Element} element
+   * @param {Number=} index fake it for testing only, will delete later
+   * @return {string}
+   * @private
+   */
+  getThumbnailUrl_(element, index) {
+    if (element.tagName == 'AMP-IMG') {
+      return element.getAttribute('src');
```

Comment:
@dvoytenko oh this part is just a quick mock to be able to focus on the gallery UI. This will fully change with the thumbnail protocol.